### PR TITLE
Writer: add show comments info into status bar.

### PR DESCRIPTION
**Pull request by GulsahKose**
Change-Id: I2d3a2b4eb17a839a2ed51e60a0e6c2c66d89b7e6

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by eszkadev — APPROVED** [review]
it works, thanks :)

**Review by pedropintosilva — APPROVED** [review]

**Comment by pedropintosilva** [comment]
@GulsahKose please run the failing test locally https://cpci.cbg.collabora.co.uk:8080/job/github_online_master_debug_vs_co-24.04_cypress_desktop/3490/console probably that additional element requires some cypress test update

```js
      cy:command ✔  cGet	#toolbar-down #prev
      cy:command ✘  click	
      cy:command ✔  fail:	
                    Test failed: integration_tests/desktop/writer/statusbar_spec.js / Statubar tests. / Switching page.
                    
                    Timed out retrying after 10050ms: `cy.click()` failed because the center of this element is hidden from view:
                    
                    `<div class="unotoolbutton jsdialog ui-content unospan-prev prev no-label" tabindex="-1" id="prev" data-cooltip="Previous Page">...</div>`
                    
                    Fix this problem, or use `{force: true}` to disable error checking.
                    
                    https://on.cypress.io/element-cannot-be-interacted-with
                    
                    /home/collabora/jenkins/workspace/github_online_master_debug_vs_co-24.04_cypress_desktop/cypress_test/integration_tests/desktop/writer/statusbar_spec.js:22:40
                      20 |         cy.cGet('body').contains('#menu-insert li a', 'Page Break').click();
                      21 |         desktopHelper.assertVisiblePage(1, 2, 2);
                    > 22 |          ...
          cy:log ✱  Finishing test: integration_tests/desktop/writer/statusbar_spec.js / Statubar tests. / Switching page.
```

**Comment by GulsahKose** [comment]
Pedro, I'm checking the fail. Failing test is trying to click "previous page" and most probably can not find the button. As i see button is still there with same id. Couldn't understand how it failing. Still investigating how my patch affected.

But i detect a problem even without my patch. The following video is recorded without my patch. Next page and previous page doesn't move the page to next or previous. 

[Screencast from 2024-12-19 13-48-18.webm](https://github.com/user-attachments/assets/bbca1f87-c545-47e0-9c2b-cff759981a8a)

File: cypress_test/integration_tests/desktop/writer/statusbar_spec.js [hunk 1]
```diff
@@ -25,9 +25,9 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Statubar tests.', function
 		cy.cGet('#menu-insert').click();
 		cy.cGet('body').contains('#menu-insert li a', 'Page Break').click();
 		desktopHelper.assertVisiblePage(1, 2, 2);
-		cy.cGet('#toolbar-down #prev').click();
```
[the remark this answers is not in the document]
Comment (GulsahKose) [reply cypress_test/integration_tests/desktop/writer/statusbar_spec.js:28]: Done thanks

**Pull request merged**